### PR TITLE
Added: Truncate statement

### PR DESCRIPTION
**Pull request by stillalivx**
From issue #206

### CODE REVIEW

File: lib/database.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ export type DatabaseOptions =
 export type SyncOptions = {
   /** If tables should be dropped if they exist. */
   drop?: boolean;
+  truncate?: boolean;
```
Comment (eveningkid) [inline lib/database.ts:39]: Would be good to have a description comment:
```js
/** If tables should be truncated. Will raise errors if the linked tables don't exist. */
```

File: lib/model.ts [hunk 2]
```diff
@@ -139,6 +139,17 @@ export class Model {
     this._isCreatedInDatabase = false;
   }
 
+  /** Truncate a model in the database */
```
Comment (eveningkid) [inline lib/model.ts:142]: Ok I know I'm annoying but there's a "." missing at the end :')

```js
 /** Truncate a model in the database. */
```

**Review by eveningkid — CHANGES_REQUESTED** [review]
All good, I'm impressed how much you know the codebase! It's perfect really :)

Just need to update the comments and we can merge this.

Thank you!

File: lib/database.ts (hunk 1 above)
Comment (stillalivx) [reply lib/database.ts:39]: Okay, on my way!

File: lib/model.ts (hunk 2 above)
Comment (stillalivx) [reply lib/model.ts:142]: Oh, didn't read this one.

**Review by eveningkid — APPROVED** [review]

**Comment by eveningkid** [comment]
Cool, thank you for being so reactive :)

Let's meeeeerge!!

**Pull request merged**